tests/plugins/enterkey/manual/emptylineselection.md

```diff
@@ -0,0 +1,24 @@
+@bender-tags: 4.16.1, bug, 4626
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, enterkey
+
+1. Place a cursor right after 'foo' in either of the editors
+
+```
+foo
+   ^ - put selection here
+```
+
+2. Press `Enter` twice.
+
+3. Press `Left` key or `Up` key or `Backspace` to move the cursor to the empty line
```

Comment:
Manual tests have to be exhaustive, ie. we want to check all these options - not just one. So at first we want to press `left` and check if styles are preserved, then go back, press `up` and check, and the third time check with `backspace` - instead of randomising test and checking only one option.